sys/include/net/loramac.h

```diff
@@ -0,0 +1,715 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_loramac LoRaMAC
+ * @ingroup     net
+ * @brief       LoRaMAC definitions
+ *
+ * @{
+ *
+ * @file
+ * @brief       LoRaMAC header definitions
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef NET_LORAMAC_H
+#define NET_LORAMAC_H
+
+#include <stdint.h>
+#include <stdlib.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LoRaMAC default parameters
+ * @{
+ */
+/**
+ * @brief   Default device EUI
+ *
+ *          8 bytes key, required for join procedure
+ */
+#ifndef LORAMAC_DEV_EUI_DEFAULT
+#define LORAMAC_DEV_EUI_DEFAULT        { 0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00 }
+#endif
+
+/**
+ * @brief   Default application EUI
+ *
+ *          8 bytes key, required for join procedure
+ */
+#ifndef LORAMAC_APP_EUI_DEFAULT
+#define LORAMAC_APP_EUI_DEFAULT        { 0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00 }
+#endif
+
+/**
+ * @brief   Default application key
+ *
+ *          16 bytes key, required for join procedure
+ */
+#ifndef LORAMAC_APP_KEY_DEFAULT
+#define LORAMAC_APP_KEY_DEFAULT        { 0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00 }
+#endif
+
+/**
+ * @brief   Default application session key
+ *
+ *          16 bytes key, only required for ABP join procedure type
+ */
+#ifndef LORAMAC_APP_SKEY_DEFAULT
+#define LORAMAC_APP_SKEY_DEFAULT       { 0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00 }
+#endif
+
+/**
+ * @brief   Default network session key
+ *
+ *          16 bytes key, only required for ABP join procedure type.
+ */
+#ifndef LORAMAC_NWKS_KEY_DEFAULT
+#define LORAMAC_NWKS_KEY_DEFAULT       { 0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00, \
+                                         0x00, 0x00, 0x00, 0x00 }
+#endif
+
+/**
+ * @brief   Default device address
+ */
+#ifndef LORAMAC_DEV_ADDR_DEFAULT
+#define LORAMAC_DEV_ADDR_DEFAULT       { 0x00, 0x00, 0x00, 0x00 }
+#endif
+/** @} */
+
+/**
+ * @name    LoRaMAC default values
+ * @{
+ */
+/**
+ * @brief   Default NetID (only valid with ABP join procedure)
+ */
+#ifndef LORAMAC_DEFAULT_NETWORK_ID
+#define LORAMAC_DEFAULT_NETWORK_ID      (1U)
```

Comment:
In the doc you name it NetID. Why not in the name (would have the benefit of being much shorter)

Reply:
I'll change that

Reply:
done